CIP-1694/README.md

```diff
@@ -0,0 +1,504 @@
+
+### Separation of Hard Fork Initiation from Standard Protocol Parameter Changes
+
+In contrast to other protocol parameter updates, hard forks (or, currently, changes to the protocol's major version) require much more attention.
+Indeed, while other protocol parameter changes can be performed without significant software changes,
+a hard fork assumes that a supermajority of the network has upgraded the node to support the new set of features that are introduced by the upgrade. This means that the timing of a hard fork event must be communicated well ahead of time to all Cardano users, and requires coordination between stake pool operators, wallet providers, DApp developers, and the node release team.
+
+Hence, this proposal promotes hard fork initiations as a standalone governance action, separate from protocol parameter updates, as in the Shelley scheme.
+
+### Treasury Withdrawals vs Project Catalyst
+
+Project Catalyst is currently one of the main drivers for withdrawals from the Cardano treasury. Each Catalyst round is usually followed by hundreds - if not thousands - of MIR requests to deliver funding to selected projects. In this new governance framework, however, we will only permit one trasury withdrawal per epoch.
+
+Since all withdrawal requests from the treasury must fit into a single transaction, this limits the number of projects that can be funded in a single epoch. If necessary, this can be worked around by e.g. splitting funding over multiple epochs, by transferring funds to a temporary holding pot, or by restricting the number of projects that are funded in each round.
+
+## Path to Active
+
+### Acceptance Criteria
+
+- [ ] A new ledger era is enabled on the Cardano mainnet, which implements the above specification. This will be split into two stages, as described above.
+
+### Implementation Plan
+
+This proposal restricts itself to the core on-chain mechanism. However, several issues remain to be discussed. These include:
+
+1. off-chain DRep expectations;
+2. bootstrapping the Constitutional Committee;
+3. the initial Constitution;
+4. DRep incentives;
+5. which values in the [ratification requirement table](#requirements) should be hard-coded, and which should be protocol parameters?
+
+Some of these issues may be addressed by changes to this document; others will need to be addressed in new CIPs or discussion documents.
+
+#### New Protocol Parameters
+
+New protocol parameters will be needed for the following:
+
+* the governance action deposit amount
+* governance action expiration (as a number of epochs from the current epoch)
+* thresholds for treasury withdrawals
```

Comment:
Historically most governments have made every effort to spend saved wealth and engage in debt or deficit spending in order to maintain political power or status. It is conceivable that an entire governance system would willfully agree to spend as much as available.

IMHO multiple spending levels are wise, however I would recommend considering another safety feature. The ADA reserve is paid out each epoch according to the parameter Rho. Unlike Bitcoin's discrete block reward halfling schedule, Cardano selected a smooth continuous reserve payout based on Rho. Rho prevents any governance decision from draining or paying out the reserve early. Similarly, the treasury is accumulated each epoch with the parameter Tau. Just as Rho protects reserve payout rate, it may be wise to use a parameter, Tau-prime, to establish the amount of additional ADA per epoch is made available for Treasury payouts. Treasury payout availability could be accumulated over many epochs. Tau-prime would prevent all of the treasury from being immediately available to any governance decision (including a greedy oligarchy) and would form an equilibrium balance with Tau governing a smooth continuous rate of treasury accumulation or depletion over time. Over many epochs and many cycles of governance decisions Rho, Tau, and Tau-prime would be foundational monetary policy parameters of the system.

Reply:
I think that makes a lot of sense. Indeed we put the three treasury withdrawal levels in place to prevent things from being drained too quickly (and with increasing threshold limits, which I realize was never explicitly stated). There is a lot of sense in these levels being proportional instead of constant!

Reply:
maybe I can resolve this conversation since I've added it to the rationale section?